orbit/cmd/desktop/desktop.go

```diff
@@ -175,7 +174,7 @@ func setupLogs() {
 
 	dir = filepath.Join(dir, "Fleet")
 
-	if err := os.MkdirAll(dir, 0755); err != nil {
+	if err := os.MkdirAll(dir, 0o755); err != nil {
```

Comment:
👍 nice! curious: was this causing problems?

Reply:
This is my linter (not sure which one) auto-formatting this.